mmdet/core/hook/wandblogger_hook.py

```diff
@@ -0,0 +1,450 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import warnings
+
+from mmcv.runner import HOOKS
+from mmcv.runner.dist_utils import master_only
+from mmcv.runner.hooks.checkpoint import CheckpointHook
+from mmcv.runner.hooks.logger.wandb import WandbLoggerHook
+
+from mmdet.core import EvalHook
+
+
+@HOOKS.register_module()
+class WandbLogger(WandbLoggerHook):
```

Comment:
We can also name this hook `WandbLoggerHook` and name `mmcv.WandbLoggerHook` `BaseWandbLoggerHook`.

Reply:
Done. ✅

Reply:
When I do this, it's throwing the error that the module is already registered.

Reply:
Oh, it's my fault. Seems the WandbLoggerHook has been registered in HOOKS in mmcv.